internal/v1/handler_blueprints_test.go

```diff
@@ -108,15 +108,6 @@ func TestHandlers_CreateBlueprint(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "Invalid blueprint name", jsonResp.Errors[0].Title)
 
-	// Test customization users, user without password and key is invalid
```

Comment:
Well what if we tested that the `user without password and key is **valid**` instead?

Reply:
Good point, I am putting all the tests back but this one I am thinking removing because it sort of breaks the flow of this test - it tries to create a record several times and then the final one is success. I think authentication is well covered by the other dozen of tests if you don't mind :-)

Rebased.